src/IonParserTextRaw.ts

```diff
@@ -1096,45 +1095,54 @@ export class ParserTextRaw {
                     }
                 }
                 break;
+            default:
+                throw new Error("can't get this value as a string");
+        }
+        return s;
+    }
+
+    get_value_as_uint8array(t: number) : Uint8Array {
+        let bytes = [];
+        switch (t) {
             case T_CLOB2:
```

Comment:
Seems like there's an opportunity to consolidate the implementations of the `T_CLOB2` and `T_CLOB3` cases a fair amount in the future.

Reply:
Potentially; leaving it up to the future for now